sickbeard/notifiers/pushover.py

```diff
@@ -166,7 +165,7 @@ def _notifyPushover(self, title, message, sound=None, userKey=None, apiKey=None,
 
         logger.log("Sending notification for " + message, logger.DEBUG)
 
-        return self._sendPushover(message, title, userKey, apiKey)
+        return self._sendPushover(message, title, sound=sound, userKey=userKey, apiKey=apiKey)
```

Comment:
missing sound paramter. (these cause userkey to be sent as sound, and apiKey to be sent as userKey)